nextflow.config

```diff
@@ -30,6 +30,7 @@ params {
     syfpeithi_score_threshold = 0.5
     mhcflurry_mhcnuggets_score_threshold = 0.426
     proc_chunk_size = 7500000
+    supported_allele_information = false
```

Comment:
Maybe something like `show/print_supported_models` would be more self explanatory to highlight that this information is not something the user has to provide, but turns on the help?

Reply:
Then I would go with show (to match the processes/modules)